packages/rps/src/components/LobbyPage.tsx

```diff
@@ -10,7 +10,12 @@ import CreatingOpenGameContainer from '../containers/CreatingOpenGameContainer';
 
 interface Props {
   openGames: OpenGame[];
-  joinOpenGame: (opponentName: string, opponentAddress: string, roundBuyIn: string) => void;
+  joinOpenGame: (
+    opponentName: string,
+    opponentAddress: string,
+    opponentOutcomeAddress: string,
```

Comment:
is `outcomeAddress` another word for `destination`?

Reply:
Yes